src/main/java/com/wanderersoftherift/wotr/commands/LootCommands.java

```diff
@@ -0,0 +1,109 @@
+                .then(Commands.argument(LOCATION_ARG, BlockPosArgument.blockPos())
+                        .then(Commands.argument(LOOT_TABLE_ARG, ResourceOrIdArgument.lootTable(context))
+                                .suggests(LootCommand.SUGGEST_LOOT_TABLE)
+                                .executes(ctx -> createChest(ctx, BlockPosArgument.getBlockPos(ctx, LOCATION_ARG),
+                                        ResourceOrIdArgument.getLootTable(ctx, LOOT_TABLE_ARG), 1))
+                                .then(Commands.argument(RIFT_TIER_ARG, IntegerArgumentType.integer(1))
+                                        .executes(
+                                                ctx -> createChest(ctx, BlockPosArgument.getBlockPos(ctx, LOCATION_ARG),
+                                                        ResourceOrIdArgument.getLootTable(ctx, LOOT_TABLE_ARG),
+                                                        IntegerArgumentType.getInteger(ctx, RIFT_TIER_ARG)))))));
+        builder.then(Commands.literal("fillAreaWithChests")
+                .then(Commands.argument(FROM_ARG, BlockPosArgument.blockPos())
+                        .then(Commands.argument(TO_ARG, BlockPosArgument.blockPos())
+                                .then(Commands.argument(LOOT_TABLE_ARG, ResourceOrIdArgument.lootTable(context))
+                                        .suggests(LootCommand.SUGGEST_LOOT_TABLE)
+                                        .executes(ctx -> createChests(ctx, BlockPosArgument.getBlockPos(ctx, FROM_ARG),
+                                                BlockPosArgument.getBlockPos(ctx, TO_ARG),
+                                                ResourceOrIdArgument.getLootTable(ctx, LOOT_TABLE_ARG), 1))
+                                        .then(Commands.argument(RIFT_TIER_ARG, IntegerArgumentType.integer(1))
+                                                .executes(
+                                                        ctx -> createChests(ctx,
+                                                                BlockPosArgument.getBlockPos(ctx, FROM_ARG),
+                                                                BlockPosArgument.getBlockPos(ctx, TO_ARG),
+                                                                ResourceOrIdArgument.getLootTable(ctx, LOOT_TABLE_ARG),
+                                                                IntegerArgumentType.getInteger(ctx,
+                                                                        RIFT_TIER_ARG))))))));
+    }
+
+    private int createChests(
+            CommandContext<CommandSourceStack> ctx,
+            BlockPos from,
+            BlockPos to,
+            Holder<LootTable> lootTable,
+            int riftTier) {
+        BoundingBox area = BoundingBox.fromCorners(from, to);
+        int result = 0;
+        for (BlockPos pos : BlockPos.betweenClosed(area.minX(), area.minY(), area.minZ(), area.maxX(), area.maxY(),
+                area.maxZ())) {
```

Comment:
```suggestion
        int result = 0;
        for (BlockPos pos : BlockPos.betweenClosed(from, to)) {
```

Maybe you could even use stream
```java
return BlockPos.betweenClosedStream(from, to)
        .mapToInt(pos->createChest(ctx, pos, lootTable, riftTier))
        .sum();
```